src/engraving/dom/text.cpp

```diff
@@ -62,12 +62,30 @@ Text::Text(EngravingItem* parent, TextStyleType tid)
     initElementStyle(&defaultStyle);
 }
 
+EngravingObject* Text::propertyDelegate(Pid id) const
+{
+    if (!parent()->isTextLineBaseSegment()) {
+        return nullptr;
+    }
+
+    switch (id) {
+    case Pid::TEXT_STYLE:
+        return parent();
+    default:
+        return nullptr;
+    }
```

Comment:
This function reads a bit clunky. For a function this simple I'd perhaps put the positive case first so it's easier to read, e.g.
```
if (id == Pid::TEXT_STYLE && parent()->isTextLineBaseSegment()) {
    return parent();
}
return nullptr;
```

Also, I think parent() _may_ be null in some cases  so perhaps nullchecking it before using it is worth it just to be safe